### ISSUE TITLE
Website Accessibility-WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (a[href$="gitlab\.matrix\.org\/"])

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1210]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
a[href$="gitlab\.matrix\.org\/"]
#### Snippet
`<a href="https://gitlab.matrix.org/" class="w-inline-block"><img src="/images/gitlab.svg" alt="" class="mxfooter__icon"></a>`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Microsoft Edge version 96.0.1054.62
